.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.16
+      - image: cimg/golang:1.18
```

Comment:
Still no luck
![image](https://user-images.githubusercontent.com/1274789/168890428-c74260fc-bc51-444b-90fd-6755b87ed8fe.png)

Reply:
Is it `cimg/go:1.18`?
An accompanying question: have you verified Docker build works overall?

Reply:
You're correct, I've validated the circleci execution from within that docker image. Is there an associated Dockerfile specifically for tfswitch I should be testing? I didn't see one in the repo.

Reply:
I was thinking of just pulling down docker image, exec'ing into it and running te commands from `.circleci/config.yml` manually 🤔 I'm not familiar with CircleCI =(
Let's leave this to @warrensbox 😉

Reply:
Ah okay, that's exactly what I did then. I thought you were referring to a non-existent docker image specifically for tfswitch.
I'm familiar with CircleCi and use it currently for work.

Reply:
Great. Thanks!